EasyPost/Exceptions/API/_Base.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Reflection;
+using EasyPost.Models.API;
+using EasyPost.Utilities;
+using RestSharp;
+
+namespace EasyPost.Exceptions.API
+{
+    public class ApiError : EasyPostError
+    {
+        public readonly string? Code;
+        public readonly List<Error>? Errors;
+        public readonly int? StatusCode;
+
+        /// <summary>
+        ///     Get a formatted error string with expanded details about the error.
+        /// </summary>
+        /// <returns>A formatted error string.</returns>
+        public string PrettyPrintString
```

Comment:
Question: Can we just call it `PrettyPrint` so it's simpler and matches other already existing langs?

We also need a unit test for this to ensure the output is correct.

Reply:
Sounds good. I didn't want it to sound like this function will actually print the string (i.e. to console); it only returns the string.

Reply:
Resolved